modules/core/src/main/java/org/apache/ignite/internal/processors/cache/mvcc/MvccUtils.java

```diff
@@ -588,32 +585,9 @@ private static boolean isVisible(GridCacheContext cctx, long link,
      */
     private static <R> R invoke(GridCacheContext cctx, long link, MvccClosure<R> clo, MvccSnapshot snapshot)
         throws IgniteCheckedException {
-        assert cctx.mvccEnabled();
```

Comment:
Do we need UnsupportedOperationException here? Otherwise, the behaviour variates depending on '-ea' JVM process option.

Reply:
We're not changing the behaviour, assert -> assert.
Just removing the unused code.
This method will be removed later as unused (when it become unused)